### DISCUSSION TITLE
why the table [derived_gas_outputs] has not update

### DISCUSSION THREAD

**Discussion by zenghao306** [filecoin-project/lily#330] [Q&A] [answered]
when i set the actorstate-workers flag just as 10 for example,why the table [derived_gas_outputs] has not update the  data for a long time?

**Comment by frrist** [comment, marked as answer]
Same answer as here: https://github.com/filecoin-project/sentinel-visor/discussions/334#discussioncomment-260959

We recently add a new command called "walk" which should work better for you.

You can run it like this to index message gas output states:

visor walk --from 1000 --to 2000 --tasks=chaineconomics 

You can include multiple tasks by comma separating them. The current list is at [master/chain/indexer.go#L23](https://github.com/filecoin-project/sentinel-visor/blob/master/chain/indexer.go#L23)
